easydock/database.py

```diff
@@ -144,15 +159,17 @@ def init_db(db_fname, input_fname, prefix=None):
     data_smi = []  # non 3D structures
     data_mol = []  # 3D structures
     for mol, mol_name in read_input.read_input(input_fname):
-        smi = Chem.MolToSmiles(mol, isomericSmiles=True)
-        if prefix:
-            mol_name = f'{prefix}-{mol_name}'
-        if mol_is_3d(mol):
-            data_mol.append((mol_name, smi, Chem.MolToMolBlock(mol)))
-        else:
-            data_smi.append((mol_name, smi))
-    cur.executemany(f'INSERT INTO mols (id, smi) VALUES(?, ?)', data_smi)
-    cur.executemany(f'INSERT INTO mols (id, smi, source_mol_block) VALUES(?, ?, ?)', data_mol)
+        isomers = get_isomers(mol)
+        for stereo_index, stereo_mol in enumerate(isomers):
+            smi = Chem.MolToSmiles(stereo_mol, isomericSmiles=True)
+            if prefix:
+                mol_name = f'{prefix}-{mol_name}'
+            if mol_is_3d(mol):
+                data_mol.append((mol_name, stereo_index, smi, Chem.MolToMolBlock(stereo_mol)))
+            else:
+                data_smi.append((mol_name, stereo_index, smi))
```

Comment:
```python 
        if prefix:
            mol_name = f'{prefix}-{mol_name}'
        if mol_is_3d(mol):
            data_mol.append((mol_name, 0, smi, Chem.MolToMolBlock(stereo_mol)))
        else:
            isomers = get_isomers(mol, max_isomers=max_isomers)
            for stereo_index, stereo_mol in enumerate(isomers):
                smi = Chem.MolToSmiles(stereo_mol, isomericSmiles=True)
                data_smi.append((mol_name, stereo_index, smi))
```

1. Enumeration is only needed if input is not a 3D molecule
2. I added `max_isomers` to input arguments